sdk/schemaregistry/Azure.Data.SchemaRegistry/README.md

```diff
@@ -42,13 +42,9 @@ The simpliest way is to use the [Azure portal][azure_portal] and navigate to you
 Once you have the Azure resource credentials and the Event Hubs namespace hostname, you can create the [SchemaRegistryClient][schema_registry_client]. You'll also need the [Azure.Identity][azure_identity] package to create the credential.
 
 ```C# Snippet:SchemaRegistryCreateSchemaRegistryClient
-string endpoint = "<event_hubs_namespace_hostname>";
-var credentials = new ClientSecretCredential(
-    "<tenant_id>",
-    "<client_id>",
-    "<client_secret>"
-);
-var client = new SchemaRegistryClient(endpoint, credentials);
+// Create a new access Spark batch client using the default credential from Azure.Identity using environment variables previously set,
+// including AZURE_CLIENT_ID, AZURE_CLIENT_SECRET, and AZURE_TENANT_ID.
+var client = new SchemaRegistryClient(endpoint: endpoint, credential: new DefaultAzureCredential());
```

Comment:
Should we link over to the Identity ReadMe and let that be the authoritative voice for configuring your environment for the credential?   That may help keep this from getting stale in the face of any Identity changes in the future.

Reply:
Not a bad idea. I copy/pasted this comment from the several other libraries that had the same thing. Also, just realized it mentions a Spark client. LOL. Forgot to actually read the comment.

Reply:
Added.